run_scans.py

```diff
@@ -22,7 +22,7 @@ def launchTestsArgs(tool, shelf, slot, link, chamber, vfatmask, scanmin, scanmax
   #Build Commands
   setupCmds = []
   preCmd = None
-  cmd = ["%s"%(tool),"-s%d"%(slot),"-g%d"%(link),"--shelf=%i"%(shelf), "--nevts=%i"%(nevts), "--vfatmask=%s"%(vfatmask)]
+  cmd = ["%s"%(tool),"-s%i"%(slot),"-g%i"%(link),"--shelf=%i"%(shelf), "--nevts=%i"%(nevts), "--vfatmask=%i"%(vfatmask)]
```

Comment:
Looks like:

https://github.com/bdorney/vfatqc-python-scripts/blob/master/run_scans.py#L198

Maybe calling `hex()` here will ensure the log file has this written in hex format.

For chamberInfo.py if we have:

`--vfatmask=0x%x`

I understood from:

https://docs.python.org/2/library/optparse.html#optparse-standard-option-types

That the number should be formatted like "0x...." but from [chamber_vfatmask](https://github.com/bdorney/vfatqc-python-scripts/blob/bc120ab04e6b20eb302585deeaf6d2bda0bd4ddf/chamberInfo.py#L37)

The inputs are `0x0` so they should instead be stored as `0` otherwise we would pass `0x0x0` to vfatmask which in the `append()` command is a string so it is not interpreted as a hex number.

Basically we need to resolve 0x... vs. 0x0x... or have I misunderstood?